pkg/operator/ceph/version/version.go

```diff
@@ -37,7 +38,7 @@ var (
 )
 
 func (v *CephVersion) String() string {
```

Comment:
I think it would be a good idea to add a new method specifically for returning a properly formatted ceph string so that we can document a specific path of machine readable information between the version detected in the operator and the parsing in the init container via ROOK_CEPH_VERSION.